data/projects/templates/default.mpt

```diff
@@ -29,7 +29,7 @@
         <bbtrack>
           <trackcontainer width="640" x="610" y="5" maximized="0" height="400" visible="0" type="bbtrackcontainer" minimized="0">
             <track muted="0" type="0" name="Kicker" solo="0">
-              <instrumenttrack pan="0" fxch="0" usemasterpitch="1" pitchrange="1" pitch="0" basenote="57" vol="100">
+              <instrumenttrack pan="0" fxch="0" usemasterpitch="1" pitchrange="1" pitch="0" basenote="69" vol="100">
```

Comment:
Why did you change this while leaving the TripleOscillator track as is? Isn't everything supposed to be handled by the upgrade routine

Reply:
I wish I could tell you, but I hardly remember what I had for lunch 2 days ago, let alone:
```7e55bfb124 (Martin Pavelek  2019-12-17 21:37:34 +0100  32)               <instrumenttrack pan="0" fxch="0" usemasterpitch="1" pitchrange="1" pitch="0" basenote="69" vol="100">```
:D

Looking at the file in the original microtuner PR, the TripleOsc track is also adjusted to 69. For some reason, the change on line 8 did not make it here. o_O

In any case, the last version of the upgrade routine should indeed take care of it, so this change is no longer needed. I will test it and push an update. Thanks.